stacks-signer/src/v0/signer.rs

```diff
@@ -1144,7 +1159,25 @@ impl Signer {
                 ),
             ))
         } else {
-            None
+            block_response
+        }
+    }
+
+    #[cfg(any(test, feature = "testing"))]
```

Comment:
Same idea here -- can this go into a dedicated test module?

Reply:
Can do :D